zerver/migrations/0698_stream_subscriber_count.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 5.1.7 on 2025-04-05 22:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("zerver", "0697_empty_topic_name_for_dms_from_third_party_imports"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="stream",
+            name="subscriber_count",
+            field=models.PositiveIntegerField(default=0),
```

Comment:
you'll want to use `db_default=0` to avoid this migration needing to rewrite every `Stream` row immediately.